pkg/bundle/additional.go

```diff
@@ -88,5 +88,13 @@ func (o *AdditionalOptions) GetAdditional(_ v1alpha1.PastMirror, cfg v1alpha1.Im
 		return nil, err
 	}
 
-	return image.AssociateImageLayers(opts.FileDir, assocMappings, images)
+	assocs, err := image.AssociateImageLayers(opts.FileDir, assocMappings, images)
+	if err != nil {
+		return nil, err
+	}
+	for _, assoc := range assocs {
+		assoc.Type = image.TypeGeneric
```

Comment:
@estroz This type value is being set on a copy of the Association and not being set in the return Associations map.

Reply:
Fixed by updating the map after setting type.